types/request/index.d.ts

```diff
@@ -280,7 +284,7 @@ declare namespace request {
         // set in `Request.prototype.init`
         headers: Headers;
         method: string;
-        pool: false | { [key: string]: http.Agent | https.Agent };
+        pool?: false | { [key: string]: http.Agent | https.Agent } & PoolOptions | PoolOptions;
```

Comment:
This `?` shouldn't be added. `pool` is always set. `Request.prototype.init` sets it to the global pool if it's not provided in the options.
https://github.com/request/request/blob/master/request.js#L171-L173

Reply:
Sure, it is my fault. I will fix to it